stdlib/public/SDK/Intents/INRequestRideIntent.swift

```diff
@@ -25,7 +25,7 @@ extension INRequestRideIntent {
     paymentMethod: INPaymentMethod? = nil,
     scheduledPickupTime: INDateComponentsRange? = nil
   ) {
-    if #available(iOS 10.3, watchOS 3.2, *) {
+    if #available(iOS 10.3, *) {
```

Comment:
The whole extension is only available in watchOS 3.2, so this check will always be true. (At least that what the compiler warning said).